roles/create_vm_template/tasks/validation/fail_if_vm_already_exists.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+
+- name: Retrieve all the details about the new VM
+  vmware.vmware_rest.vcenter_vm_info:
+    #vmware.vmware_rest.vcenter_vm:
+    vcenter_hostname: "{{ connection_args.vcenter_hostname }}"
+    vcenter_username: "{{ connection_args.vcenter_username }}"
+    vcenter_password: "{{ connection_args.vcenter_password }}"
+    vcenter_validate_certs: "{{ connection_args.vcenter_validate_certs }}"
+    names:
+    - "{{ vm_name }}"
+  register: vms
+
+
+- name: Fail when the VM is already exists
+  ansible.builtin.fail:
+    msg: The VM with the same name already exists. Please provide a different VM name
```

Comment:
We should not fail, but pass, and update if needed.

Reply:
The validation shoule be in a different role as I understood - I will remove it for now